database/migrations/2024_06_26_132929_create_default_presentations_table.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('default_presentations', function (Blueprint $table) {
+            $table->id();
+
+            $table->string('name')->nullable();
+            $table->text('description')->nullable();
+            $table->time('start');
+            $table->time('end');
+            $table->string('type')
+                ->comment('Can contain only opening or closing');
```

Comment:
This can be implemented with an enum: `$table->enum('type', ['opening', 'closing']);`. Maybe next time?